README.md

```diff
@@ -147,6 +147,45 @@ resources:
         newKey: user_id
 ```
 
+### Batch Resources with `properties` paramerters
+
+Instead of accepting just a list of users (`user_ids`), a batch resouce could accept both a list of users (`user_ids`) and a list of properties (`properties`):
```

Comment:
```suggestion
Instead of accepting just a list of users (`user_ids`), a batch resource could accept both a list of users (`user_ids`) and a list of properties (`properties`) to fetch about that user:
```

Reply:
Thanks for catching the spell errors! I installed the code spell checker on VS, it will help me in the future.